src/particles/wakefields/WakeConvolution.H

```diff
@@ -0,0 +1,33 @@
+#ifndef WAKE_CONVOLUTION_H
+#define WAKE_CONVOLUTION_H
+
+#include "../ImpactXParticleContainer.H"
+#include "../../initialization/InitDistribution.H"
+
+using namespace amrex;
+
+//Constants
+constexpr double e = 1.602e-19;  // Elementary charge [C]
+constexpr double c = 3.0e8;  //Speed of light [m/s]
+constexpr double m_e = 9.109e-31;  //Electron mass [kg]
```

Comment:
The same goes for many of these constants:
[](https://warpx.readthedocs.io/en/23.11/_static/doxyhtml/namespaceablastr_1_1constant_1_1_s_i.html)